### PR TITLE
Fix envrionments adding configure APM function

### PR DESCRIPTION
**Pull request by 0xGabi**
# 🦅 Pull Request

Solves #1090 

Fix interaction with all environments. 

Use a `configureAPM` function to setup the global apm option.

### CODE REVIEW

File: packages/cli/src/lib/environment/configEnvironment.js [hunk 1]
```diff
@@ -93,12 +118,11 @@ export function configEnvironment({
     : null
 
   return {
-    apm: {
-      'ens-registry': env.registry,
-      ...merge(env.apm || {}, apm || {}),
-    },
-    // if there is no arapp.json and the command is not init default to the "global" config
-    // designed for the dao commands including dao acl
+    apm: configureAPM(
```
Comment (0xGabi) [inline packages/cli/src/lib/environment/configEnvironment.js]: Here is where the magic happen 🔮

File: packages/cli/src/lib/environment/configEnvironment.js [hunk 2]
```diff
@@ -30,6 +34,25 @@ function getEnv(arapp, environment) {
   }
 }
 
+const configureAPM = (ipfsRPC, gateway, ensRegistryAddress) => {
```
Comment (dapplion) [inline packages/cli/src/lib/environment/configEnvironment.js]: Please add JSDocs to this function to known the data structures of `ipfsRPC, gateway, ensRegistryAddress`. If not full at least parcial :pray:

Comment (0xGabi) [reply packages/cli/src/lib/environment/configEnvironment.js]: Let's do it after release

Comment (dapplion) [reply packages/cli/src/lib/environment/configEnvironment.js]: Sounds good

**Pull request merged**